lib/l10n_utils/fluent.py

```diff
@@ -5,16 +5,30 @@
 from django.core.cache import caches
 from django.utils.encoding import force_bytes
 
-from fluent.runtime import FluentLocalization, RootedFileResourceLoader
-
-from lib.l10n_utils import translation
+from fluent.runtime import FluentLocalization, FluentResourceLoader
 
 
 cache = caches['l10n']
 
 
+class FluentL10n(FluentLocalization):
+    def has_message(self, message_id):
+        # assume English locales have the message
+        if self.locales[0].startswith('en-'):
+            return True
```

Comment:
Do you want en-GB to pass here? But yes, I think this would be better suited for a subclass.

One question I have is if you want this to return true for English, or for in-development. If just for English is good enough, this could have an option "including_fallback=False", which would skip the `break` and the locale check.

Reply:
We'll probably need to return True for development as well, that's a good point. I'm not sure yet whether to return `False` for `en-US` for example. I think the majority of FTL files for `en-*` are going to be very short and the `en` fallback for whatever it is will be good enough for the code block to be shown. This is probably something I just need to discuss with the dev team.